### DISCUSSION TITLE
Project Repo Links

### DISCUSSION THREAD

**Discussion by metasean** [USU-ITLS-x265-23-summer/class-repo#132] [Show and tell]
Once your project has been approved:
1. Use this [Project Template](https://github.com/USU-ITLS-x265-23-summer/project) to start your project.
2. You'll need to "Use this Template" (big green button, toward the top center)
3. Once your repo is created, post your project repo link as a thread in this discussion.
4. To use your repo:
  - follow the tips in the `README.md`
  - ask, and share, tips with your classmates

**Comment by derekscholes** [comment]
https://github.com/derekscholes/derekscholes.github.io

I just updated this repo to fit everything I have added. My other major thing I need to work on is CSS. There is a lot of CSS on each page which is now becoming confusing. I think I need to create a CSS file for each web page so there is a bit more organization.

**Reply by metasean** [reply]
> I think I need to create a CSS file for each web page so there is a bit more organization.

Remember that CSS stands for "_cascading_ style sheets".  The cascade works in a variety of ways.

Review the following, and they should help you figure out how to organize your stylesheets:
- https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade
- https://css-tricks.com/css-cascade-layers/
- https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance

**Reply by metasean** [reply]
@derekscholes
I'm having some permission issues trying to spin up the CodeSpaces for your repo.  
Please (a) give me, and preferably also your classmates, access ***OR*** (b) use the [course's project template](https://github.com/USU-ITLS-x265-23-summer/project).  Thank you!

**Reply by derekscholes** [reply]
I just gave everyone access as collaborators. I will take a look at the information you sent because I really need to organize it better. It is very confusing at the moment.

**Comment by jakegros** [comment]
@metasean I don't seem to be able to access the Project Template.  Is it found somewhere else?

**Reply by jakegros** [reply]
This is what I get when I try to access the project template online from computer

![template error](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133597504/8ff44d3c-0c25-4c47-a721-bfd7763270aa)

When I try from my phone app, it says "Could not resolve to a Repository with the name 'USU-ITLS-x265-23-summer/project'."
Any thoughts?

**Reply by kensnyder** [reply]
That is the correct link. GitHub will sometimes do that if you aren't
logged in or if your current user has no access to it.

From your screenshot it appears you are logged in. Do you have two accounts
maybe? I can add both accounts to the group if it makes it easier.

**Reply by jakegros** [reply]
This is the only account I have.  It is like I don't have write access or something.

**Reply by kensnyder** [reply]
@jakegros Try now. I re-added you to the class list.

**Comment by Skye404** [comment]
Here is the link to my project repo, using the template: https://github.com/USU-ITLS-x265-23-summer/Skye404Project/tree/main 

My project doesn't show up in my own personal repo. Do I need to add it somehow to my own page? @kensnyder 

I think I made it private. Do I need to make it public or add everyone's names somehow?

**Reply by kensnyder** [reply]
@Skye404 Yes, you'll want it be public.

To change it from private to public, go to the very bottom of the settings
page <https://github.com/USU-ITLS-x265-23-summer/Skye404Project/settings>
and click the first item in the "Danger Zone".

**Comment by kensnyder** [comment]
That is the correct link. GitHub will sometimes do that if you aren't
logged in or if your current user has no access to it.

From your screenshot it appears you are logged in. Do you have two accounts
maybe? I can add both accounts to the group if it makes it easier.

**Reply by Skye404** [reply]
I don't believe I have two accounts in GitHub. I only created this one for this class, so I'm not sure why it doesn't show up in my own repo.

I made my project public now.

**Reply by kensnyder** [reply]
Oh sorry, That message about the two accounts was supposed to be for Jake. I just replied about your repo question on the correct thread.

**Comment by kensnyder** [comment]
Yes, you'll want it be public.

To change it from private to public, go to the very bottom of the settings
page <https://github.com/USU-ITLS-x265-23-summer/Skye404Project/settings>
and click the first item in the "Danger Zone".

**Comment by jakegros** [comment]
Thank you, Ken.  That worked.  How can this be at this point?

On Mon, Jul 10, 2023 at 3:36 PM Ken Snyder ***@***.***> wrote:

> @jakegros <https://github.com/jakegros> Try now. I re-added you to the
> class list.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/132#discussioncomment-6409729>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A73ISQFA26YYAIK7X7RKXRTXPRYURANCNFSM6AAAAAAZKLW3BM>
> .
> You are receiving this because you were mentioned.Message ID:
> <USU-ITLS-x265-23-summer/class-repo/repo-discussions/132/comments/6409729@
> github.com>
>


--

**Reply by kensnyder** [reply]
I know. Weird.

**Comment by coopsand** [comment]
https://github.com/USU-ITLS-x265-23-summer/coopsandportfolio/tree/main

**Reply by kensnyder** [reply]
Yes @coopsand I can access your repository.
